node/narwhal/committee/src/prop_tests.rs

```diff
@@ -133,3 +196,27 @@ fn committee_members(input: CommitteeInput) {
     // (2f + 1) + (f + 1) - 1 = 3f + 1 = N
     assert_eq!(quorum_threshold + availability_threshold - 1, total_stake);
 }
+
+#[proptest]
+fn invalid_stakes(#[strategy(too_low_stake_committee())] committee: Result<Committee<CurrentNetwork>>) {
+    assert!(committee.is_err());
+    if let Err(err) = committee {
+        assert_eq!(err.to_string().as_str(), "All members must have sufficient stake");
+    }
+}
+
+#[proptest]
+fn invalid_member_count(#[strategy(too_small_committee())] committee: Result<Committee<CurrentNetwork>>) {
+    assert!(committee.is_err());
+    if let Err(err) = committee {
+        assert_eq!(err.to_string().as_str(), "Committee must have at least 4 members");
+    }
+}
+
+#[proptest]
+fn invalid_round(#[strategy(invalid_round_committee())] committee: Result<Committee<CurrentNetwork>>) {
+    assert!(committee.is_err());
+    if let Err(err) = committee {
+        assert_eq!(err.to_string().as_str(), "Round must be nonzero");
+    }
```

Comment:
You might be able to do:
```suggestion
    assert!(matches!(committee, Err(e) if e.to_string().as_str() == "Round must be nonzero"))
```